cypress/integration/Case4_Menu.js

```diff
@@ -93,7 +97,23 @@ describe('Test case 4: Check functionality', () => {
       })
   })
 
-  // it('Check Add App icon popup', () => {
-  //   HomePage.optionsButton.click()
-  // })
+  it('Check sending Magic Link', () => {
+    HomePage.optionsButton.should('be.visible')
+    cy.wait(100) //whithout this "wait" cypress clicks on element when it is not attachecd to DOM yet
```

Comment:
i'm not sure i understand why, how can it both be visible and not attached to DOM?I think the animation should stop when feed is ready.

Reply:
This is what i see in Cypress log
![2020-01-21_16-50-51](https://user-images.githubusercontent.com/47723945/72814864-75e17380-3c6e-11ea-88d0-7e233c3ed72e.png)